.changeset/silly-chefs-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro-integration-kit": minor
+---
+
+Exposes a new `hmrIntegration` from `astro-integration-kit`. It's now recommended inside your local playground to get development HMR instead of using `watchIntegration`
```

Comment:
```suggestion
Exposes a new `hmrIntegration` integration from `astro-integration-kit`. It's now recommended inside your local playground to use development HMR instead of using the `watchIntegration` utility
```

Should "get" be "use"? I'm not 100% sure what this means, apologies!